release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -244,6 +244,117 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
 existing CRD storage versions are not missing in the upgraded CRD, avoiding
 potential data loss.
 
+[id="ocp-4-5-operator-api"]
+==== Read-only Operator API (Technology Preview)
+
+The new Operator API is now available as a Technology Preview feature in
+read-only mode. Previously, installing Operators using Operator Lifecycle
+Manager (OLM) required cluster administrators being aware of multiple APIs,
+including CatalogSources, Subscriptions, ClusterServiceVersions, and
+InstallPlans. This new, single Operator API resource is a first step towards a
```

Comment:
Small nit but "This single new Operator API resource..." sounds less clunky to me.

Reply:
Just dropping "new" completely; I was borderline about it sounding redundant anyway.